test-apps/ui-test-app/src/common/rpcs.ts

```diff
@@ -3,32 +3,16 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import {
-  IModelReadRpcInterface, IModelTileRpcInterface, IModelWriteRpcInterface, RpcInterfaceDefinition,
+  IModelReadRpcInterface, IModelTileRpcInterface, RpcInterfaceDefinition,
   SnapshotIModelRpcInterface,
 } from "@bentley/imodeljs-common";
 import { PresentationRpcInterface } from "@bentley/presentation-common";
-import { Config } from "@bentley/bentleyjs-core";
 
 /**
  * Returns a list of RPCs supported by this application
  */
 export function getSupportedRpcs(): RpcInterfaceDefinition[] {
 
-  if (Config.App.has("imjs_TESTAPP_ALLOW_WRITE") && (Config.App.get("imjs_TESTAPP_ALLOW_WRITE") === "1")) {
```

Comment:
Need to update README.md. @bbastings uses ui-test-app for testing editing tools. Does it still support editing standalone iModels?

Reply:
Yes, ui-test-app lets you edit standalone imodels (needed to test toolsettings/tool assistance)